layouts/header.html

```diff
@@ -68,9 +66,10 @@
         <div class="collapse navbar-collapse" id="navbar">
           <ul class="nav navbar-nav navbar-right main-nav">
             <li><a href="/docs/introduction/overview/">Docs</a></li>
+            <li><a href="/download/">Download</a></li>
             <li><a href="/community/">Community</a></li>
             <li><a href="/blog/">Blog</a></li>
-            <li><a href="https://github.com/prometheus"><i class="fa fa-github"></i> Github</a></li>
+            <li><a href="https://github.com/prometheus"><i class="fa fa-github"></i></a></li>
```

Comment:
Since this is now only an icon using up less space, how about also adding a Twitter icon:

```
<li><a href="https://twitter.com/PrometheusIO"><i class="fa fa-twitter"></i></a></li>
```